src/rosetta-helpers.ts

```diff
@@ -446,6 +531,193 @@ export function getOptionsFromOperations(operations: RosettaOperation[]): Rosett
   return options;
 }
 
+function parseStackingContractCall(
```

Comment:
The code coverage is also showing that none of the conditions in this function are ever ran